src/@orb.yml

```diff
@@ -80,13 +87,10 @@ jobs:
           attach-workspace: true
           repo: <<parameters.ecr_repo>>
           tag: <<parameters.tags>>
+          dockerfile: <<parameters.dockerfile>>
       - run:
           name: "install kubectl"
```

Comment:
stage name does not fit action. maybe do separate step `install-tools` and `rollout`

Reply:
fixed